release/audacity.xml

```diff
@@ -22,12 +25,13 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 -->
-<cleaner id="audacity" os="linux">
+<cleaner id="audacity">
   <label>Audacity</label>
   <description>fast, cross-platform audio editor</description>
   <option id="autosave">
     <label>Autosaved files</label>
     <description>Autosaved files for session restoration</description>
     <action search="walk.files" command="delete" path="~/.audacity-data/autosave/"/>
+    <action search="walk.files" command="delete" path="$APPDATA\Audacity\AutoSave\*.autosave"/>
```

Comment:
Any .autosave file. These are left behind on system/app hang/crashes
if autosave is enabled. Rare, but Windows managed it with me. Of
course, I have never seen anything else in there, so purging any item
in there might be okay.

On 6/20/16, Andrew Ziem <notifications@github.com> wrote:
>>    <label>Audacity</label>
>>    <description>fast, cross-platform audio editor</description>
>>    <option id="autosave">
>>      <label>Autosaved files</label>
>>      <description>Autosaved files for session restoration</description>
>>      <action search="walk.files" command="delete"
>> path="~/.audacity-data/autosave/"/>
>> +    <action search="walk.files" command="delete"
>> path="$APPDATA\Audacity\AutoSave\*.autosave"/>
>
> Did you mean `search="glob"` or to remove `*.autosave`?
>
> ---
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub:
> https://github.com/az0/cleanerml/pull/109/files/212a4dbfa5658c96d849ac6ef367534572446c8a#r67799506